app/src/main/res/values-sw600dp-port/dimens.xml

```diff
@@ -438,4 +438,21 @@
   <dimen name="administrator_controls_download_permissions_view_auto_update_topic_constraint_layout_padding_start">16dp</dimen>
   <dimen name="administrator_controls_download_permissions_view_auto_update_topic_constraint_layout_padding_end">16dp</dimen>
   <dimen name="administrator_controls_download_permissions_view_auto_update_topic_widget_switch_compat_margin_end">4dp</dimen>
+
+  <!-- AdministratorControlAccountActionsView  -->
+  <dimen name="administrator_control_account_actions_view_account_actions_text_view_padding_start">16dp</dimen>
+  <dimen name="administrator_control_account_actions_view_account_actions_text_view_padding_end">16dp</dimen>
+  <dimen name="administrator_control_account_actions_view_log_out_text_view_padding_start">16dp</dimen>
+  <dimen name="administrator_control_account_actions_view_log_out_text_view_padding_end">16dp</dimen>
```

Comment:
I guess we don't need to introduce duplicate entries from values/dimens.xml to other files. Because, in case these values are not here, it would automatically reference to the same variable under values/dimens.xml. Ditto for other places.